### PR TITLE
add options attribute, bpmn can be customized

### PR DESCRIPTION
**Pull request by meterXu**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "vue-bpmn",
-  "version": "0.2.2",
+  "version": "0.2.3",
```
Comment (pinussilvestrus) [inline package.json:3]: Let's not update the package version manually. We do that when we release via `np` in an orderly manner.

File: src/index.vue [hunk 2]
```diff
@@ -22,10 +25,10 @@
       var container = this.$refs.container;
 
       var self = this;
-
-      this.bpmnViewer = new BpmnJS({
+      const _options = Object.assign({
         container: container
-      });
+      },this.options)
```
Comment (pinussilvestrus) [inline src/index.vue:30]: Until we don't have linting in place for this project: let's add some space here

> }, this.options)

File: src/index.vue [hunk 3]
```diff
@@ -22,10 +25,10 @@
       var container = this.$refs.container;
 
       var self = this;
-
-      this.bpmnViewer = new BpmnJS({
+      const _options = Object.assign({
```
Comment (pinussilvestrus) [inline src/index.vue:28]: we mix `const` and `var` here. Let's not do this.

File: src/index.vue [hunk 4]
```diff
@@ -11,6 +11,9 @@
       url: {
         type: String,
         required: true
+      },
+      options:{
+        type:Object
```
Comment (pinussilvestrus) [inline src/index.vue:16]:
```suggestion
      options: {
        type: Object
```

**Review by pinussilvestrus — COMMENTED** [review]
Thanks for your contribution 👏 

Before we can merge this can you make sure to
* update your commit(s) to a single one fitting our [commit message guidelines](https://www.conventionalcommits.org/)

> feat: add options

* sign the [CLA](https://github.com/bpmn-io/vue-bpmn/pull/8#issuecomment-802686484)

Besides, I'd love to see a minimal example how this new configuration feature could be used in a Vue application. Can you maybe provide some by updating the `README`.md or the [example](https://github.com/bpmn-io/vue-bpmn/tree/master/example)?

File: src/index.vue (hunk 4 above)
Comment (nikku) [reply src/index.vue:16]: We should consider documenting this, too.

File: package.json (hunk 1 above)
Comment (meterXu) [reply package.json:3]: ok

Comment (meterXu) [reply package.json:3]: Okay, you can refer to this example
https://github.com/bpmn-io/bpmn-js-examples/tree/master/properties-panel-extension

File: README.md [hunk 5]
```diff
@@ -22,6 +23,15 @@ Use [bpmn-js](https://github.com/bpmn-io/bpmn-js) to display BPMN 2.0 diagrams i
...
     },
+    data() {
+      return {
+        options: {
+          propertiesPanel: {},
+          additionalModules: [],
+          moddleExtensions: []
+        }
+      }
+    }
```
Comment (nikku) [inline README.md:34]: Missing `,`.

**Review by nikku — CHANGES_REQUESTED** [review]
Looking good, overall.

**Pull request merged**